frappe/public/js/frappe/form/grid_row.js

```diff
@@ -995,6 +987,13 @@ export default class GridRow {
 				if (frappe.ui.form.editable_row !== me) {
 					var out = me.toggle_editable_row();
 				}
+
+				// Set dependant property for current row
+				Object.keys(me.columns).forEach((column) => {
+					me.set_dependant_property(me.columns[column].df);
+				});
+				me.refresh();
```

Comment:
this can potentially cause a looot of refresh :eyes:

Reply:
Hmm, need to re-render the row otherwise it doesn't update properly